src/jit/codegenxarch.cpp

```diff
@@ -470,8 +470,7 @@ void CodeGen::genSetRegToConst(regNumber targetReg, var_types targetType, GenTre
             if (*(__int64*)&constValue == 0)
             {
                 // A faster/smaller way to generate 0
-                instruction ins = genGetInsForOper(GT_XOR, targetType);
-                inst_RV_RV(ins, targetReg, targetReg, targetType);
+                inst_RV_RV(INS_xorps, targetReg, targetReg, targetType);
```

Comment:
You should fix the comment above that says "xorpd"

Reply:
Done. Funny that the comment was already incorrect, before this change the JIT generated either `xorps` or `xorpd` depending on the target type.